decentraland/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "scripts": {
-    "start": "FORCE_COLOR=1 node starter.js",
+    "start": "FORCE_COLOR=1 CONFIG=development node starter.js",
```

Comment:
Yes, make sense. Since the `helpers/starter.js` should be ES5 right now because it isn't using babel. Should we do it right now or extract to a `tech debt` issue?

Reply:
Yep, makes sense. `tech debt` issue is fine with me!

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/272
Resolving this.